templates/paloalto_panos_show_interface_hardware.textfsm

```diff
@@ -1,8 +1,9 @@
 Value INTF (\S+)
-Value SPEED (\d+|\[n\/a\])
+Value ID (\S+)
```

Comment:
Do you mind adding additional `.raw` and `.yml` to the tests folder for this change?

Reply:
Yes, see https://github.com/networktocode/ntc-templates/pull/680